packages/manager/modules/anthos/src/dashboard/ips/remove-private-ip/remove-private-ip.html

```diff
@@ -0,0 +1,27 @@
+<form name="removePrivateIpForm">
+    <oui-modal
+        data-heading="{{:: 'anthos_tenant_dashboard_ips_remove_private_ip_modal_title' | translate }}"
+        data-primary-action="$ctrl.onRemovePrivateIpClick()"
+        data-primary-label="{{:: 'anthos_tenant_dashboard_ips_remove_private_ip_modal_action_remove' | translate }}"
+        data-primary-disabled="$ctrl.privateIp.inUse"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{:: 'anthos_tenant_dashboard_ips_remove_private_ip_modal_action_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isRemoving"
+    >
+
+        <!--Description-->
+        <div>
```

Comment:
useless div

Reply:
I prefered to have a div for a set of things.
Here div enclosed content for description ant related extra IP info

Reply:
may be you can have a <section> for this kind of use cases?